### ISSUE TITLE
Mockup: digital student card + mensa credit

### ISSUE THREAD

**Issue by kordianbruck** [TCA-Team/TumCampusApp#1009]
We want to create an activity that displays one’s own student card in the app and shows the mensa credit on the card. As we first need to talk to the Studentenwerk, we will first need a Mockup / XML in Android, that will showcase the usecase and make it clear that certain aspects are highlighted to the user:

- The credit is not "live" and can be lagging behind the real credit on the card. This needs to be made very clear
- There is no liability from our side, that the money shown is correct
- The digital student card is not a valid replacement for the analog version

**Comment by thellmund** [comment]
A first version below. The blue background still feels off. 

<img width="323" alt="digital studentcard 01" src="https://user-images.githubusercontent.com/11819826/43150866-49036a64-8f6b-11e8-9d06-23c34361712e.png">
<img width="288" alt="digital studentcard 02" src="https://user-images.githubusercontent.com/11819826/43150867-4937e14a-8f6b-11e8-9f8f-3994b44900f2.png">

**Comment by kordianbruck** [comment]
@xsrf any comments from your side? Can we go to the studentenwerk with this? Should we separate the "card" more from the credit?

**Comment by xsrf** [comment]
Hmm... C-Number? Expiration date? "TUM" in the center looks just wrong, please move it to the right top according to the style-guide. Showing the credit combined with the virtual card is fine, I guess.
If STWM is not okay with showing the expiration date, you could just show it if it expires before the enrolled semester ends.

**Comment by thellmund** [comment]
@xsrf What is the “c-number”? Also, is there an additional expiration date besides the date until which the student is enrolled at the university? If so, could you provide more detail?

I will move the logo to the right and see how that looks 👍

**Comment by pfent** [comment]
@thellmund I think c-number is the card number
Those cards also have an expiry date, but we probably would need to query TUMonline / STWM for that date.

**Comment by xsrf** [comment]
> Those cards also have an expiry date, but we probably would need to query TUMonline / STWM for that date.

Yeah. That's why you made a mockup that you want to present to STWM ;)

**Comment by kordianbruck** [comment]
ISIC also has a digital student card: https://www.isic.de/de/partnerbereich/isic-gutzuwissen/isic-anerkennung/

So we are not dreaming up requirements here :)

**Comment by kordianbruck** [comment]
@thellmund on which branch can I find this mockup?
